### PR TITLE
Add Hypathia documentation

### PR DESCRIPTION
**Pull request by gonzaponte**
This PR adds the documentation for the city Hypathia.
The built documentation can be found [here](https://pr010-sw-docs.readthedocs.io/en/latest/hypathia.html).

### CODE REVIEW

**Review by jmbenlloch — APPROVED** [review]
Correct documentation for Hypathia. It is a simple city similar to Irene and everything is well described in this PR. Thanks!

File: docs/source/hypathia.rst [hunk 1]
```diff
@@ -1,5 +1,156 @@
...
+ * ``/RD/sipmrd``
+
+.. _Hypathia output:
+
+Output
+------
+
+Same as :doc:`irene`:
+
+ * ``/PMAPS/S1``: the sliced PMT-summed waveform for each S1 peak. 4 columns: event number, peak number, time (:math:`\mu`\ s) and amplitude (pes)
```
Comment (martinperezmaneiro) [inline docs/source/hypathia.rst:32]: Here we have 5 columns, bwidth is missing

File: docs/source/hypathia.rst [hunk 2]
```diff
@@ -1,5 +1,156 @@
...
+ * ``/PMAPS/S1Pmt``: the sliced individual PMT waveforms for each S1 peak. 4 columns: event number, peak number, pmt id and amplitude (pes)
+ * ``/PMAPS/S2``: the sliced PMT-summed waveform for each S2 peak. 4 columns: event number, peak number, time (:math:`\mu`\ s) and amplitude (pes)
```
Comment (martinperezmaneiro) [inline docs/source/hypathia.rst:34]: Same here

**Review by martinperezmaneiro — APPROVED** [review]
Besides that little comment, the documentation for Hypathia is clear and understandable, thanks!

**Pull request merged**